### DISCUSSION TITLE
More Date/Time details

### DISCUSSION THREAD

**Discussion by mounirhnf** [community/community#126249] [Mobile]
![Screenshot_20240528-161047.jpg](https://github.com/community/community/assets/58731883/890615ac-8fa1-4107-a4f7-43a4e02e1dfc)

It would be great if we could see the exact time and date of a specific commit in the Mobile app instead of an abbreviation.

**Comment by taki-on** [comment]
@mounirhnf would you mind telling me if it happens on iOS or Android?

**Reply by mounirhnf** [reply]
I am currently using Android, I can't tell you about IOS though. So as far as I know it's happening on Android

**Reply by tfry-git** [reply]
This probably applies across the board. Definitely also to the desktop site: https://github.com/orgs/community/discussions/122848
